lib/std/compress/zstandard.zig

```diff
@@ -0,0 +1,274 @@
+const std = @import("std");
+const Allocator = std.mem.Allocator;
+
+const types = @import("zstandard/types.zig");
+
+const RingBuffer = @import("zstandard/RingBuffer.zig");
+pub const decompress = @import("zstandard/decompress.zig");
+pub usingnamespace @import("zstandard/types.zig");
```

Comment:
Please avoid `usingnamespace` entirely. I'm still not 100% sure it will make it into the language after all.

Reply:
Done - running `rg usingnamespace` is showing a number of other places where it's being used that might want a a followup issue.